dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.jsp

```diff
@@ -1,29 +1,51 @@
 <%@ page import="com.liferay.portal.language.LanguageUtil" %>
 <div id="${id}" class="fileAjaxUploader">
+    <input name="${name}" id="${name}ValueField" dojoAttachPoint="fileNameField" type="hidden" value="${fileName}" />
+    <div dojoAttachPoint="fileUploadForm" class="fileAjaxUploaderForm">
+        <input
+            name="${name}FileUpload"
+            type="file"
+            accept="${accept}"
+            dojoAttachEvent="onchange:_doFileUpload"
+            dojoAttachPoint="fileInputField"
+            class="form-text"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)